### DISCUSSION TITLE
__init__ arguments error

### DISCUSSION THREAD

**Discussion by fotoente** [yupix/Mi.py#57] [report bug] [answered]
Hello,

When running the Misskey Advent Calender code as provided (only things i changed is the URL and the AuthToken) i get the Error that __init__ don't get the right arguments.

`Traceback (most recent call last):
  File "/home/misskey/ceilingfox-bot/autoreply/reply.py", line 24, in <module>
    bot = MyBot('!')
  File "/home/misskey/ceilingfox-bot/autoreply/reply.py", line 8, in __init__
    super().__init__(cmd_prefix)
TypeError: object.__init__() takes exactly one argument (the instance to initialize)`

**Comment by yupix** [comment, marked as answer]
Ah ... I understand the cause. Please change the code of the following blog as follows

```diff
-class MyBot(commands.Cog):
+class MyBot(commands.Bot):
    def __init__(self, cmd_prefix: str):
        super().__init__(cmd_prefix)
```

**Reply by yupix** [reply]
I wrote an article in a hurry, so it seems that I made a mistake. Sorry for the inconvenience
